packages/components/psammead-bulletin/src/index.jsx

```diff
@@ -16,14 +20,100 @@ import { getPica, getLongPrimer } from '@bbc/gel-foundations/typography';
 import { mediaIcons } from '@bbc/psammead-assets/svgs';
 import VisuallyHiddenText from '@bbc/psammead-visually-hidden-text';
 import { Link, LiveLabel } from '@bbc/psammead-story-promo';
+import { grid } from '@bbc/psammead-styles/detection';
+
+const twoOfSixColumnsMaxWidthScaleable = `33.33%`;
+// (2 / 6) * 100 = 0.3333333333 = 33.33%
+
+const fourOfSixColumnsMaxWidthScaleable = `66.67%`;
+// (4 / 6) * 100 = 66.6666666667 = 66.67%
+
+const fullWidthColumnsMaxScaleable = `100%`;
+// (12 / 12) * 100 = 100 = 100%
+
+const halfWidthColumnsMaxScaleable = `50%`;
+
+const bulletinWrapperStyles = `
+  position: relative;
+  background-color: ${C_LUNAR};
+  display: grid;
+  grid-template-columns: repeat(6, 1fr);
+  grid-column-gap: ${GEL_SPACING_DBL};
+`;
+
+const imageWrapperStyles = `
+  vertical-align: top;
+  display: inline-block;
+  grid-column: 1 / span 6;
+  @supports (${grid}) {
+    width: initial;
+  }
+`;
+
+const textWrapperStyles = `
+  grid-column: 1 / span 6;
+  display: inline-block;
+  width: ${fullWidthColumnsMaxScaleable};
+  @supports (${grid}) {
+    width: initial;
+    padding: 0;
+  }
+`;
+
+const TVBulletinWrapper = styled.div`
+  ${bulletinWrapperStyles};
+  padding: ${GEL_SPACING};
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
+    padding: ${GEL_SPACING_DBL};
+  }
+`;
+
+const TVImageWrapper = styled.div`
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
+    grid-column: 1 / span 3;
+    width: ${halfWidthColumnsMaxScaleable};
+  }
+  ${imageWrapperStyles};
+`;
 
-const BulletinWrapper = styled.div`
+const TVTextWrapper = styled.div`
   @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
-    display: none;
+    grid-column: 4 / span 3;
+    width: ${halfWidthColumnsMaxScaleable};
+    ${({ dir }) =>
+      dir === 'ltr'
+        ? `padding-right: ${GEL_SPACING_DBL};`
```

Comment:
```suggestion
        ? `padding-left: ${GEL_SPACING_DBL};`
```
As this is right now it's causing the padding on IE to be on the wrong side
<img width="956" alt="Screenshot 2019-11-22 at 10 50 51" src="https://user-images.githubusercontent.com/12303856/69407631-0b679200-0d16-11ea-8c6b-185dd58eada1.png">